src/components/counter-container/CounterContainer.tsx

```diff
@@ -0,0 +1,87 @@
+import {StyleSheet, Theme, useTheme} from '@atb/theme';
+import {AccessibilityProps, View, ViewStyle} from 'react-native';
+import {ThemeText} from '@atb/components/text';
+import React from 'react';
+import {iconSizes} from '@atb-as/theme';
+import {TextNames} from '@atb/theme/colors';
+
+interface StylingCounter {
+  padding: number;
+  lineHeight: number;
+  type: TextNames;
+}
+export const CounterContainer = ({
+  count,
+  size = 'normal',
+  style,
+  ...a11yProps
+}: {
+  count: number;
+  size?: keyof Theme['icon']['size'];
+  style?: ViewStyle;
```

Comment:
It is preferred to use `StyleProp<ViewStyle>` instead of just `ViewStyle`. I know it is not consistent in the app from before, but I'm just commenting to raise awareness :) 

The difference with using `StyleProp` is that the parameter works more as the `style` parameter on the "native" React components. For example `style={[style.first, style.second]}` will only work if using `StyleProp`.